starfyre/main_template.py

```diff
@@ -0,0 +1,76 @@
+def get_main_template(user_routes, path):
```

Comment:
same for this function

Reply:
@DeleMike 👋 

These comments need to be addressed too 😄

Reply:
Done